ui/app/AppLayouts/Wallet/views/RightTabView.qml

```diff
@@ -29,7 +29,7 @@ Item {
         function getBackButtonText(index) {
             switch(index) {
                 case 1:
-                return qsTr("Assets")
+                return qsTr("Collectibles")
```

Comment:
design says assets , I do think thats wrong but can we double check with someone from Design?

https://www.figma.com/file/FkFClTCYKf83RJWoifWgoX/Wallet-v2?node-id=6455%3A126137&t=5AeHPRDxszbKxt7N-4

Reply:
Oh that's true! I missed that on the design, I'll check with them

Reply:
John confirmed it should be "Collectbiles"